config/gen/bot.py

```diff
@@ -148,6 +148,7 @@
     subscribe_to_hub_on_start=''
     if is_simulation():
         xbee_serial_port='/tmp/xbeebot' + str(bot_index)
+        subscribe_to_hub_on_start='subscribe_to_hub_on_start { hub_id: 1 modem_id: ' + str(common.comms.xbee_modem_id(common.comms.hub_node_id)) + ' changed: true }'
```

Comment:
Now that we're generating an inventory.yml for simulation, this line should no longer be necessary (as the hub will initiate all the subscriptions). Can you try testing without this line?